arch/risc-v/src/common/espressif/Make.defs

```diff
@@ -233,6 +236,7 @@ ifeq ($(CONFIG_ESP_WIRELESS),y)
 	$(Q) echo "Applying patches..."
 	$(Q) cd chip/$(ESP_HAL_3RDPARTY_REPO)/components/mbedtls/mbedtls && git apply ../../../nuttx/patches/components/mbedtls/mbedtls/*.patch
 endif
+endif
```

Comment:
I think we should have a "else" here to instruct to user to download the repository manually, otherwise it will fail silently and the end user will not know what to do. @fdcavalcanti @hartmannathan @eren-terzioglu suggestions?